packages/space-opera/src/components/model_viewer_snippet/components/validation.ts

```diff
@@ -166,6 +183,18 @@ export class Validation extends ConnectedLitElement {
     }
   }
 
+  countJoints(gltf: GLTF) {
+    const jointSet = new Set();
+    if (gltf.skins != null) {
+      for (const skin of gltf.skins) {
```

Comment:
Do we care to have report.info when gltf.skins is null? If so, we could probably have an early return when gltf.skins is null

And, the three dots concept is cool. Didn't know about it. Though not sure what happens when this.report doesn't exist

Reply:
It looks nicer in the output when it says `0 joints` instead of `joints`, so I switched it to this way. 

I also love the ellipsis, and I'm also not sure what happens when the original in undefined, but I went ahead and  initialized `this.report = {}` to avoid that.